Sugar/Sugar.Nougat.OSX.oxygene

```diff
@@ -11,7 +11,7 @@
     <ProjectGuid>{ab7ab88b-2370-43bf-844b-54d015da9e57}</ProjectGuid>
     <AssemblyName>Sugar</AssemblyName>
     <DefaultUses>Foundation</DefaultUses>
-    <DeploymentTargetVersion>10.6</DeploymentTargetVersion>

```

Comment:
As discussed via e-mail, NSRegularExpression only appeared in version 10.7 of OS X, and @dwarfland gave the green light to bump up the minimum targeted version.